src/containers/AdminContainer/AdminContainer.js

```diff
@@ -1,15 +1,147 @@
+        });
+        event.preventDefault();
+    }
+
+    showNewEmergency = () => {
+        return (
+            <div>
+                <p>Report a new emergency. Make sure all details are correct before publishing.</p>
+                <form onSubmit={this.handleNewSubmit}>
+                    <div class="inputGroup">
+                        <label for="title">Title</label>
+                        <input type="text" name="title" id="title"/>
+                    </div>
+                    <div class="inputGroup">
+                        <label for="description">Description</label>
+                        <input type="text" name="description" id="description"/>
+                    </div>
+                    <div class="inputGroup">
+                        <label for="contactName">Contact Name</label>
+                        <input type="text" name="contactName" id="contactName"/>
+                    </div>
+                    <div class="inputGroup">
+                        <label for="contactEmail">Contact Email</label>
+                        <input type="email" name="contactEmail" id="contactEmail"/>
+                    </div>
+                    <div class="inputGroup">
+                        <label for="contactPhone">Contact Phone</label>
+                        <input type="tel" name="contactPhone" id="contactPhone"/>
+                    </div>
+                </form>
+            </div>
+        );
+    }
+
+    showEmergencies = () => {
+        // TODO : determine how to display all current emergencies, and allow admin
+        // to choose one to update
+        return (
+            <div></div>
+        );
+    }
+
+    showUpdateEmergency = () => {
         return (
             <div>
-                admin stuff
+                <p>Update an existing emergency.</p>
+                <form onSubmit={this.handleUpdateSubmit}>
+                    <div class="input-group">
+                        <label for="update">Update</label>
+                        <input type="text" name="update" id="update"/>
+                    </div>
+                </form>
             </div>
         );
     }
+
+    render() {
+        const newEmergency = this.showNewEmergency();
+        const updateEmergency = this.updateEmergency();
+        if (this.state.userData && isAdmin(this.state.userData)) {
+            // Render page if user is logged in, and is an admin
+            return (
+                <div>
+                    <p>Either create a new emergency, or update an existing emergency</p>
+                    <div>
+                        <a onClick={this.createNewEmergency}><div>Create New Emergency</div></a>
+                        <a onClick={this.updateEmergency}><div>Update Emergency</div></a>
+                    </div>
+                    <div>
+                        {this.state.createEmergency && showNewEmergency}
+                        {this.state.updateEmergency && showEmergencies}
```

Comment:
I think the variables above on lines 119 and 120 are misnamed, if I'm understanding what you're trying to do correctly.

Reply:
Yeah, some typos, should be fixed now.